PythonVisualizations/OrderedArray.py

```diff
@@ -19,9 +19,9 @@ class OrderedArray(VisualizationApp):
     nextColor = 0
 
     def __init__(self, size=10, title="Ordered Array", **kwargs):
-        kwargs['title'] = title
-        super().__init__(**kwargs)
+        super().__init__(title=title, **kwargs)
         self.size = size
+        self.title = title
```

Comment:
The `title` attribute is set by superclass constructor, so line 24 is redundant.